requirements/base.txt

```diff
@@ -6,3 +6,4 @@ requests>=1.0.0
 tornado>=4.2.1
 # Required by Tornado to handle threads stuff.
 futures>=2.0
+pycurl>=7.19.3
```

Comment:
We need to have a larger discussion here before adding another base dependency to salt.

@thatch45, @meggiebot, @dmurphy18, @UtahDave